docs/jwst/linearity/description.rst

```diff
@@ -4,12 +4,16 @@ Description
 Assumptions
 -----------
 It is assumed that the input science exposure data for near-IR instruments
-have had the superbias subtraction applied, therefore the correction
-coefficients stored in the linearity reference files for those instruments
-must have been derived from data that also had the zero group subtracted.
+have had the :ref:`superbias <superbias_step>` subtraction step applied,
+therefore the correction coefficients stored in the linearity reference files
+for those instruments must have been derived from data that has also been
+bias subtracted.
+MIRI data, on the other hand, do not receive bias subtraction
+(see :ref:`calwebb_detector1 <calwebb_detector1>`) and hence the linearity
+correction is derived and applied in the raw data space.
```

Comment:
The part that could cause confusion is stating that the correction is derived and applied in "raw data space."  What does "raw data space" mean in this context?  Maybe just replace this with the equivalent from the NIR text.  E.g., "..and hence the linearity correction is derived from data that has not been bias subtracted."

Reply:
Agreed. I was having a hard figuring out the proper words to use to express something like "absolute DN" (as opposed to DN's above bias). I'll make the change you suggest.

Reply:
Fixed in most recent commit.